graphql-tool-idl-plugin/src/test/java/io/ballerina/graphql/idl/client/IDLClientGenPluginNegativeTests.java

```diff
@@ -64,6 +64,12 @@ public void testInvalidConfigName() throws IOException, InterruptedException {
         Assert.assertNull(matchingFiles);
     }
 
+    @Test(description = "function level client declaration")
+    public void testFunctionLevelClientDeclaration() throws IOException, InterruptedException {
+        File[] matchingFiles = getMatchingFiles("project_09");
+        Assert.assertNull(matchingFiles);
```

Comment:
If possible, shall we check the given error message `ERROR [main.bal:(2:5,2:11)] 'client' qualifier not allowed` here ? 🙂